js/modules/Louisa.js

```diff
@@ -0,0 +1,109 @@
+export default {
+    state: {
+        profile: {
+            username: "your-github-username",
+            fullName: "Your Full Name",
```

Comment:
Put your full name here.

```suggestion
            fullName: "Louisa Orobia Silva",
```